contracts/stabilizer/TribeReserveStabilizer.sol

```diff
@@ -103,8 +106,21 @@ contract TribeReserveStabilizer is ITribeReserveStabilizer, ReserveStabilizer {
         _tribe.setMinter(newMinter);
     }
 
+    // Transfer held TRIBE first, then mint to cover remainder
     function _transfer(address to, uint256 amount) internal override {
-        _mint(to, amount);
+        uint256 _tribeBalance = balance();
```

Comment:
this is using the cached `token` variable from the contract state, while the `_mint()` call uses a fresh `ITribe(address(tribe()))`. Not sure if there could be some bad interaction here, but it would be nice to use the same source of information for both ?

Reply:
Fixed, although it should be fine because token is immutably set to equal tribe();